app/assets/templates/team.html

```diff
@@ -0,0 +1,24 @@
+<div class="container">
+  <div class="row text-center">
+    <div class="col-sm-3">
+      <p><img src="assets/Gerry400.jpg" class="img-circle team"/></p>
+      <p class="lead "><strong>Gerry Pass</strong></p>
```

Comment:
Class name should be representative of what it's doing. Also, random space here.